digitalpy/core/persistence/persistent_object_proxy.py

```diff
@@ -1,4 +1,4 @@
-from digitalpy.model.persistent_object import PersistentObject
+from digitalpy.core.persistence.persistent_object import PersistentObject
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module